packages/myst-to-typst/tests/cite.yml

```diff
@@ -23,4 +23,4 @@ cases:
                 - type: cite
                   label: jon85
     typst: |-
-      #cite("cockett2015", "jon85")
+      #cite(<cockett2015>, <jon85>)
```

Comment:
This should just be `@cockett2015 @jon85` without a group around it, there are no citation groups in typst as far as I can tell, and the cite function only take one arg.